flowey/flowey_lib_hvlite/src/_jobs/cfg_common.rs

```diff
@@ -113,17 +113,10 @@ impl SimpleFlowNode for Node {
             ));
 
             {
```

Comment:
The fact we needed to make this oss tweak in order to fix our closed-source pipelines is a hint that we should hoist up this bit of configuration into the closed-source codebase.

In particular - we'll want a closed-source `cfg_common_msft.rs` file, which has this ADO specific logic.

* * *

More generally - flowey's architecture has come a long way since I first wrote this `cfg_common` thing, and it can most likely get refactored into a pipeline-level helper function, vs. being a separate _job. But that's for a future PR...

Reply:
Ok - if you have some time maybe you can take this one over?